1-exercises/A-this/exercise-2.js

```diff
@@ -11,7 +11,7 @@ let person = {
     name: "Maggie",
     age: 26,
     currentAddress: "London",
-    changeAddress: /* todo */
+    changeAddress: function (newAddress){ return this.currentAddress}/* todo */
```

Comment:
Agree with Mohammed. You just want it to change the address, no need to return anything in this case.